tracker/view/login.py

```diff
@@ -6,41 +6,135 @@
 from flask_login import logout_user
 from werkzeug.exceptions import Unauthorized
 
+from config import SSO_ADMINISTRATOR_GROUP
+from config import SSO_ENABLED
+from config import SSO_GUEST_GROUP
+from config import SSO_NEW_USER_DEFAULT_PASSWORD
+from config import SSO_REPORTER_GROUP
+from config import SSO_SECURITY_TEAM_GROUP
 from config import TRACKER_PASSWORD_LENGTH_MAX
 from config import TRACKER_PASSWORD_LENGTH_MIN
+from tracker import oauth
 from tracker import tracker
 from tracker.form import LoginForm
 from tracker.model.user import User
 from tracker.user import user_assign_new_token
 from tracker.user import user_invalidate
 
+from ..model.enum import UserRole
+
 
 @tracker.route('/login', methods=['GET', 'POST'])
 def login():
-    if current_user.is_authenticated:
-        return redirect(url_for('tracker.index'))
+    if SSO_ENABLED:
+        return redirect(url_for('tracker.list_user'))
```

Comment:
Shouldn't this redirect to `tracker.sso_auth`?

Reply:
I wanted the redirection to run into the authentication guard which then starts the SSO flow. Any protected route will work